docs/bo_service_migration_guide.md

```
@@ -0,0 +1,316 @@
+    retry_attempts=3,
+    api_key=None,  # Optional
+)
+```
+
+---
+
+## Handling the Deprecation Warning
+
+When using local mode, you'll see:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)